espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/DeviceInfoHelper.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.helpers
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.Build
+import android.provider.Settings.Secure
+import android.telephony.TelephonyManager
+import android.util.DisplayMetrics
+import android.view.Display
+import android.view.WindowManager
+
+class DeviceInfoHelper(private val context: Context) {
+
+    private val defaultDisplay: Display?
+        get() {
+            val windowManager = context.getSystemService(Context.WINDOW_SERVICE)
+            return if (windowManager == null) null else (windowManager as WindowManager).defaultDisplay
```

Comment:
return (windowManager as? WindowManager)?.defaultDisplay

Reply:
This file is still Java, for now. I had to change some things in it to make it work with Kotlin.

Reply:
Nvm... I was in the wrong branch